### ISSUE TITLE
not working after reboot on Dell XPS15

### ISSUE THREAD

**Issue by blurp999** [EmixamPP/linux-enable-ir-emitter#49]
Hi. 
Your script is working properly in the current session. I can see the ir-leds blinking and facial recognition ends successfully. But after reboot it doesn't work. I have to restart the program manually. I have the latest version and Your fix https://github.com/EmixamPP/linux-enable-ir-emitter/wiki/Issues doesn't work for me. `:~$ linux-enable-ir-emitter boot enable` freezes and i have to strike Ctrl-C and launch "fix config" to make everything work again. Until restart...
I work on Dell XPS-15, Ubuntu 21.04. 

**linux-enable-ir-emitter boot status :**
> 
```linux-enable-ir-emitter.service - enable the infrared emitter
Loaded: loaded (/lib/systemd/system/linux-enable-ir-emitter.service; enabled; vendor preset: enabled)
Active: inactive (dead) since Thu 2021-10-14 21:30:38 CEST; 38min ago
Process: 1828 ExecStartPre=/sbin/modprobe uvcvideo (code=exited, status=0/SUCCESS) Process: 1832 ExecStart=/usr/bin/linux-enable-ir-emitter run (code=exited, status=0/SUCCESS)
Main PID: 1832 (code=exited, status=0/SUCCESS)
paź 14 21:30:37 aga-XPS systemd[1]: Starting enable the infrared emitter...
paź 14 21:30:38 aga-XPS systemd[1]: linux-enable-ir-emitter.service: Succeeded.
paź 14 21:30:38 aga-XPS systemd[1]: Finished enable the infrared emitter.
```
>
**journalctl -xeu linux-enable-ir-emitter.service :**
```
:~$ journalctl -xeu linux-enable-ir-emitter.service
-- Boot 2229e250c9b743bb8a9071900f0fe8d5 --
paź 14 21:30:37 aga-XPS systemd[1]: Starting enable the infrared emitter.>
░░ Subject: A start job for unit linux-enable-ir-emitter.service has begun exec>
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ A start job for unit linux-enable-ir-emitter.service has begun execution.
░░ 
░░ The job identifier is 134.
paź 14 21:30:38 aga-XPS systemd[1]: linux-enable-ir-emitter.service: Succ>
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ The unit linux-enable-ir-emitter.service has successfully entered the 'dead'>
paź 14 21:30:38 aga-XPS systemd[1]: Finished enable the infrared emitter.
░░ Subject: A start job for unit linux-enable-ir-emitter.service has finished s>
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ A start job for unit linux-enable-ir-emitter.service has finished successful>
░░ 
░░ The job identifier is 134.
```

Have no idea how to make this script working after reboot.

**Comment by EmixamPP** [comment]
The command `fix config` erase the configuration. So you need to `configure` again your infrared emitter after a reboot (the emitter should desn't work). 

Ubuntu has a simlink between `/usr/lib/systemd/system/` and `/lib/systemd/system/` ? Because my installer.sh install the service in the first one but the boot status show the second one. Can you check if it is the case ? 

Maybe there are a conflict between the two same services ?

**Comment by EmixamPP** [comment]
In other words :
1. remove linux-enable-ir-emitter.service in /usr/lib/systemd/system/ and 
2. apply the wiki fix but in /lib/systemd/system/ not in /usr/lib/systemd/system/ 
3. keep my in touch if it works for you :)

**Comment by blurp999** [comment]
Thanks for your replies. At last i found a moment to take care of it.
I reconfigured the program (` sudo linux-enable-ir-emitter configure`) and checked  presence of `linux-enable-ir-emitter.service` in both directories ( /lib/systemd/system/ & /usr/lib/systemd/system/) - it was not in the latter. 
So I copied it then to both folders and made `sudo linux-enable-ir-emitter boot enable` - with no effect after reboot.
I put it in then into /usr/lib/systemd/system/ only , and then in /lib/systemd/system/ - with the same result.
